.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
     "**/.yarn": true,
     "**/.pnp.*": true
   },
-  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js"
+  "prettier.prettierPath": "./webapp/.yarn/sdks/prettier/index.js",
+  "eslint.workingDirectories": [{ "mode": "auto" }]
```

Comment:
what is it for?

Reply:
That is a Teto's recomendation to not see an error in imports